index.js

```diff
@@ -275,6 +275,160 @@ Client.prototype.ballots = function (viewingId) {
   return count
 }
 
+Client.prototype._totalContribution = function (viewingIds) {
```

Comment:
what is the entry point that users of this module are going to access? it should be the function that doesn't start with a "_" and is defined between `ballots` and `vote`, the other functions should be defined between `_updatedRules` and `_roundtrip`